src/components/photoViewer/PhotoViewer.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const PhotoViewer: React.FunctionComponent = () => {
+    return (
+      <div>
+        PhotoViewer
+      </div>
+    );
+  };
```

Comment:
These lines seem to be indented by 2 spaces too many!
```suggestion
  return (
    <div>
      PhotoViewer
    </div>
  );
};
```